docs/decisions/0005-toggle-life-expectancy.rst

```diff
@@ -0,0 +1,111 @@
+
+.. _set of use cases for toggles: https://github.com/edx/open-edx-proposals/blob/c2d3b2a/oeps/oep-0017-bp-feature-toggles.rst#use-cases
+.. _Details a reporting requirement: https://github.com/edx/open-edx-proposals/blob/c2d3b2a/oeps/oep-0017-bp-feature-toggles.rst#req-12-report
+.. _PR discussion about toggle_use_cases: https://github.com/edx/edx-platform/pull/24815#issuecomment-681174018
+.. _PR to implement this proposal: https://github.com/edx/code-annotations/pull/49
+
+Decision
+--------
+
+In order to make the toggle life expectancy more clear, and the justification
+for permanent toggles more clear, we will take the following actions:
+
+* As originally proposed, introduce ``toggle_life_expectancy`` with values of "temporary" or "permanent". This makes the options very clear.
+* Replace ``toggle_use_cases`` with a free-form ``toggle_permanent_justification``.
+
+  * ``toggle_permanent_justification`` would be required for "permanent" toggles.
+  * ``toggle_target_removal_date`` would be required for "temporary" toggles (as was the case with the earlier version of "temporary" toggles).
+  * The value would be a string description justifying why the toggle is marked as permanent.
+  * The purpose of the annotation would be very clear, and will hopefully deter declaring a toggle as "permanent" that really could be "temporary".
+  * Authors could reference use cases as required to provide justification.
+  * Legacy permanent toggles could use a value like: "Seems permanent, but toggle defined before justification was required." This would make it easier to tell the difference between a toggle marked "permanent" with a strong or weak justification.
+  * This annotation would **not** be included in the published documentation, but used for reporting purposes only.
+  * "Permanent" toggles with a weak justification could still be candidates for a DEPR(ecation) process to either remove the toggle or strengthen the justification.
+
+* We can remind authors to include use cases in the ``toggle_description`` as well, if it aids documentation for operators or other audiences.
+
+Consequences
+------------
+
+* Toggle life expectancy and justification should be more clear.
+* Toggle use cases can be noted in other free text toggle annotations, rather than having its own annotation.
+* If a stakeholder wanted to review our "permanent" toggles to determine which really must be permanent, and which could potentially be deprecated/removed:
+
+  * They might see ``toggle_permanent_justification`` with any of the following example values:
+
+    * `See ADR detailing why this toggle must be permanent: ...`
+    * `See OSPR that added this feature and explains why it is needed for different Open edX instances: ...`
+    * `Tried DEPR-XXX, but got push back because ...`
+    * `None`
```

Comment:
Is this supposed to indicate a missing justification entry? [EDIT: Moved from commit comment to PR comment.]

Reply:
I addressed with a comment in the doc.  See if it works for you.
Also, I added a commit to fix the bullet points in the docs.